draft-pda-protocol.md

```diff
@@ -887,36 +795,7 @@ for s in range(0,1000,1):
         print(bits, p, GF(p).multiplicative_generator(), b, factor(s))
 ~~~
 
-### Key encapsulation
```

Comment:
The idea here is that we use HPKE and thus don't need to specify key encapsulation beyond a reference to https://datatracker.ietf.org/doc/draft-irtf-cfrg-hpke/?

Reply:
No, we'll need to specify this.